workflows/metabolomics/mfassignr/CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```

Comment:
```suggestion

```

Reply:
This is rendered directly as e.g. https://iwc.galaxyproject.org/workflow/chipseq-pe-main/#version-history, no need to have that be included.